starter-code/index.html

```diff
@@ -40,5 +73,6 @@
     Enjoy new music every Monday
     with your own personal playlist.
     Or sit back and enjoy Radio.
+    
   </body>
```

Comment:
please try to do more on this lab when you have time and let me know if you need help